src/pint/scripts/tcb2tdb.py

```diff
@@ -0,0 +1,56 @@
+"""PINT-based tool for converting TCB par files to TDB."""
+
+import argparse
+
+from loguru import logger as log
+
+import pint.logging
+from pint.models import get_model
+from pint.models.tcb_conversion import convert_tcb_to_tdb
+
+pint.logging.setup(level="INFO")
+
+__all__ = ["main"]
+
+
+def main(argv=None):
+    parser = argparse.ArgumentParser(
+        description="PINT tool for converting TCB par files to TBD.",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument("input_par", help="Input par file name (TCB)")
+    parser.add_argument("output_par", help="Output par file name (TDB)")
+
+    args = parser.parse_args(argv)
+
+    welcome_message = """This script converts TCB par files to TDB.
+    Please note that this conversion is not exact and the timing model 
+    should be re-fit to the TOAs. 
+
+    The following parameters are converted to TDB:
+        1. Spin frequency, its derivatives and spin epoch
+        2. Sky coordinates, proper motion and the position epoch
+        3. DM, DM derivatives and DM epoch
+        4. Keplerian binary parameters
+    
+    The following parameters are NOT converted although they are 
+    in fact affected by the TCB to TDB conversion:
+        1. Parallax
+        2. TZRMJD and TZRFRQ
+        3. DMX parameters
+        4. Solar wind parameters
+        5. Binary post-Keplerian parameters including Shapiro delay 
+           parameters
+        6. Jumps and DM Jumps
+        7. FD parameters
+        8. EQUADs
+        9. Red noise parameters including FITWAVES, powerlaw red noise and 
+           powerlaw DM noise parameters
+    """
+    log.warning(welcome_message)
```

Comment:
I think this should only be shown when run with `--help` or similar.  But a separate warning about the accuracy should be part of the conversion method, not the script.

Reply:
I have changed this to a help message. 

A brief warning will be emitted from `convert_tcb_to_tdb` function every time it is called.